app/views/public_recipes/index.html.erb

```diff
@@ -0,0 +1,21 @@
+<h1 class="text-center fw-bold">Public Recipes</h1>
+<div class="public-recipes">
+  <% if @public_recipes.present? %>
+    <% @public_recipes.each do |recipe| %>
+      <div class="border shadow py-3 px-5 m-5 row bg-white">
+        <div class="col-5">
+          <h2 class="mb-3"><%= recipe.name %></h2>
+          <h3>By: <%= recipe.user.name %></h3>
+        </div>
+        <div class="col-7">
+          <p>Total food items: <%= recipe.foods.count %></p>
+          <p>Total price: $<%= recipe.foods.sum { |food| food.price * food.quantity }  %></p>
+        </div>
+      </div>
+    <% end %>
+  <% else %>
+    <div class="d-flex mt-5 pt-5 text-secondary">
+      <h2>You have no public recipes</h2>
+    </div>
+  <% end %>
+</div>
```

Comment:
- [ ] It will be great to ensure that you show the creator of the recipe in your public recipe. As you can see in the [wireflame](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/recipe-app/images/public_recipe_index.png?token=GHSAT0AAAAAACGHAT3FPPQ5VVQIIASTDRGWZH43XEQ) the creator of the recipe is shown. However, in your application, it seems the name of the creator is not yet pulled in. Kindly fix this.
- [ ] It will be great to ensure that when you click on a recipe it navigates to the recipe show page. As you can see below, This is a major requirement. ⬇️ 
![Screenshot 2023-09-09 at 23 10 34](https://github.com/matee-safi/Rails-recipe-app/assets/32361963/3d164175-8144-4df0-97d6-a52992f761f9)

Reply:
- Well resolved 👍🏽 
<img width="1431" alt="Screenshot 2023-09-09 at 23 03 56" src="https://github.com/matee-safi/Rails-recipe-app/assets/58692590/4bf974a8-3406-45df-9e94-e7f6729ad515">

Reply:
Thank you